src/partials/products.html

```diff
@@ -0,0 +1,36 @@
+<section class="products">
+    <h3><span class="">100% NATURAL</span><br> PRODUCTS</h3>
+    <ul>
```

Comment:
lista powinna być siatką elementów, czyli posiadać klasę grid, a każdy element listy to grid__item. Coś jak w konspekcie zajęcia 13. Blok grid :)